pkg/cli/kubernetes/kubernetes.go

```diff
@@ -117,6 +121,30 @@ func EnsureNamespace(ctx context.Context, client k8s.Interface, namespace string
 	return nil
 }
 
+// DeleteNamespace delete the specified namespace.
+func DeleteNamespace(ctx context.Context, client k8s.Interface, namespace string) error {
```

Comment:
Can you change this to a non-exported function unless there's a need to call it elsewhere? We want to have most code use the interface so we can test it.